packages/terra-button/tests/wdio/button-spec.js

```diff
@@ -1,14 +1,4 @@
 /* global browser, Terra, before */
-const shouldTheme = (customProperties, selector) => {
```

Comment:
Nice. Glad this is finally gone :)

Reply:
I'll get rid of the terra-form-radio on another pull request

Reply:
Actually that one is different and needs to stay for now. The toolkit implementation will apply each variable one at at time and take a screenshot for each change. To get the correct theming screenshots for form-radio, multiple variables need to change at one time due to the default being browser naive.